src/shogun/classifier/svm/NewtonSVM.cpp

```diff
@@ -48,7 +49,7 @@ CNewtonSVM::~CNewtonSVM()
 }
 
 
-bool CNewtonSVM::train_machine(CFeatures* data)
+void CNewtonSVM::init_model(CFeatures* data)
 {
 	if (data)
```

Comment:
i think there should be a check/assertion that features are dense float64 somewhere

Reply:
is this happening anywhere?

Reply:
as discussed check for dot features is enough